static/app/views/settings/components/showNewToken.tsx

```diff
@@ -0,0 +1,66 @@
+import {MouseEventHandler} from 'react';
+import styled from '@emotion/styled';
+
+import Alert from 'sentry/components/alert';
+import {Button} from 'sentry/components/button';
+import FieldGroup from 'sentry/components/forms/fieldGroup';
+import PanelItem from 'sentry/components/panels/panelItem';
+import TextCopyInput from 'sentry/components/textCopyInput';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+function ShowNewToken({
```

Comment:
Thank you for abstracting this out. Should we rename it to something like  `AuthTokenContainer` or something. There's no action happening here so starting with a verb is weird.

Reply:
Yup, let me update!